Data/Aeson/Types/ToJSON.hs

```diff
@@ -213,18 +213,18 @@ genericLiftToEncoding opts te tel = gToEncoding opts (To1Args te tel) . from1
 --
```

Comment:
(perhaps right under "A type that can be converted to JSON"; github does not let met add a comment there..)

> Instances in general /must/ specify 'toJSON' and /should/ (but do not need to) specify 'toEncoding'.